docs/beta/plugins/discord.md

```diff
@@ -1,5 +1,248 @@
+
+- **Channel**:
+
+A Discord channel, e.g. [#start-here](https://discord.gg/qadM9u) {SB: link needs checking. Got link by clicking 'Create Invite' on the channel, but getting 'invalid link' msg for some reason when I try it locally). A channel node will be connected to all messages in that channel. 
+
+Found this in the code and pasting here in case the plugin exposes these channel types. 
+
+```
+ * Discord Channels can be of various types, defined below.
+ * See: https://discordapp.com/developers/docs/resources/channel#channel-object-channel-types
+ */
+export type ChannelType =
+  | "GUILD_TEXT"
+  | "DM"
+  | "GUILD_VOICE"
+  | "GROUP_DM"
+  | "GUILD_CATEGORY"
+  | "GUILD_NEWS"
+  | "GUILD_STORE";
+```
+
+- **Reaction**:
+
+A Discord emoji reaction, e.g. [insert cute emoji reaction code]. When a user reacts to a message, a reaction node is created that is connected to the author of the reaction (user that reacted) and the message reacted to. 
+
+
+[INSERT SCREENSHOT REACTIONS TO MESSAGE]
+
+
+### Edges
+
+The Discord plugin creates the following kinds of edges (connections between nodes) 
+in the contribution graph:
+
+- **Authors**:
+
+An "authors" edge connects an author (user or bot) to a message they created. 
+
+- **References**: 
+
+{SB: below is just copied and pasted from teh Discourse draft, as I'm not sure references count in discord}
+
+```
+A references edge connects a topic or post to another referencable node (i.e. a node that corresponds to a specific url on Discourse ). {SB: taken from GitHub plugin, still correct? Just say referenceable nodes are topics/posts/users?}
+
+If the reference is pointing to a user, we call it a "mention", but from SourceCred's perspective it's the same kind of edge. {SB: taken from GitHub plugin, still correct?}
+
+References in Discourse can be either hyperlinks to referencable nodes, or quoted parts of other posts {SB: I'm kind of guessing here, would be good to get more detailed explanation, and perhaps include a illustrative example (e.g. a quote that links to both text and image if supported?). This brings up another questions I've had for a while: when you select and click the "Quote" button in the UI, it automatically creates a reply, presumably creating both a reply edge and reference edge. But what if you've already started a reply, and are selecting quotes from the UI from a post different than the one you started replying to? What if you're quoting multiple posts in the same reply?}
+```
+
+- **Reply to**:
+
+{SB: below is just copied and pasted from teh Discourse draft, as I'm not sure references count in discord}
+
+```
+A reply-to edge connects a post to the post it is replying to.  
+
+
+The Discourse plugin creates the following kinds of edges (connections between nodes) 
+in the contribution graph:
+```
+- **Creates Reaction**:
+
+A "creates reaction" edge connects a reaction node to its author and reacted to message. {SB: just guessing here}
+
+## Configuration
```

Comment:
This is solid, but I'm wary about having two sources of truth for these instructions. There's also similar setup instructions in the template-instance repo. I feel like any content that is duplicated there should be removed from this doc and replaced with a link to the template-instance README, but we can keep some of the more specific extra stuff like custom emojis here and have the template-instance README link to this doc for more specific instructions.

Reply:
So the whole configuration section is basically a rewrite of the `template-instance` repo README. Was actually wondering how best to handle this from a Cred perspective, and thought about adding you as a co-author on the PR, but figured you've already got Cred from template-instance? Let me know if that's not right though or you have other ideas. 

I actually think we should keep this content here in the docs, remove it from template-instance. We have this beautiful docs site now, and most of this is user-facing (if technical) documentation. This type of doco generally lives in a docs site if one exists. GitHub is mostly used for developer-facing docs. Typically dev docs are on the docs site too, unless there's no proper place to put them (separate dev docs site or section). Open to changing my mind though:)